packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -1471,6 +1471,52 @@ void main() {
     expect(items.take(8), orderedEquals(<int>[0, 1, 2, 3, 4, 5, 6, 7]));
   });
 
+  // Regression test for https://github.com/flutter/flutter/issues/88191
+  testWidgets('Do not crash when dragging with two fingers simultaneously ', (WidgetTester tester) async {
```

Comment:
Looks good and thanks for including a test. However, as this should be in `test/widgets/reorderable_list_test.dart` instead of the material test as this testing the base widget's functionality and isn't specific to the material ReorderableListView.

Reply:
You are right!
Will move it.